src/dotty/tools/dotc/transform/FullParameterization.scala

```diff
@@ -176,6 +176,12 @@ trait FullParameterization {
           } else EmptyTree
         }
         tree match {
+          case Return(expr, from) =>
+            val rewired = rewiredTarget(from, derived)
+            if (rewired.exists)
+              Return(expr, Ident(rewired.termRef))
+            else
+              EmptyTree
```

Comment:
1. That's a silent failure. Should at least issue `ctx.error`
2. From could be an EmptyTree(which means return from enclsoing method, see doc comment in Return).
`rewiredTarget(NoSymbol, derived)` doesn't make sens.
 Based on this I would better check if `from.isEmpty` and only if nonempty rewire `from`